addOns/pscanrules/src/main/resources/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties

```diff
@@ -362,18 +362,18 @@ pscanrules.usernameidor.soln = Use per user or session indirect object reference
 
 pscanrules.viewstate.content.email.desc = The following emails were found being serialized in the viewstate field:
 pscanrules.viewstate.content.email.name = Emails Found in the Viewstate
-pscanrules.viewstate.content.email.pattern.source = Email pattern - http://www.regular-expressions.info/regexbuddy/email.html
+pscanrules.viewstate.content.email.pattern.source = Email pattern - https://www.regular-expressions.info/regexbuddy/email.html
```

Comment:
This is a 404, https://www.regular-expressions.info/email.html seems to be the new URL.

Reply:
No worries, it's easy to overlook these things sometimes, we've all been there.

Reply:
Should be email rather than examples.